src/platform/platform_posix.c

```diff
@@ -141,32 +156,95 @@ CxPlatSystemLoad(
 
     CXPLAT_FREE(ProviderFullPath, QUIC_POOL_PLATFORM_TMP_ALLOC);
 
-#ifdef DEBUG
-    CxPlatform.AllocFailDenominator = 0;
-    CxPlatform.AllocCounter = 0;
-#endif
+    QuicTraceLogInfo(
+        PosixLoaded,
+        "[ dso] Loaded");
```

Comment:
`dso`?

Reply:
Dynamic shared object. On windows user mode this logs dll, and kernel logs sys, so dso makes the most sense for posix as its a common name on linux and macos.